<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDialog</class>
 <widget class="QDialog" name="MainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <widget class="QLabel" name="txtHeader">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>IRANSans</family>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnExit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>IRANSans</family>
    </font>
   </property>
   <property name="text">
    <string>خروج</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>210</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>IRANSans</family>
     <pointsize>10</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>جنسیت</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkAgreement">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>251</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>IRANSans</family>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>صحت موارد بالا را برسی کردم</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSubmit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>550</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>IRANSans</family>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ثبت اطلاعات</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressHealth">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>410</y>
     <width>191</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>IRANSans</family>
     <bold>false</bold>
    </font>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>180</y>
     <width>161</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="radioMale">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>IRANSans</family>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>مرد</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioFemale">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>IRANSans</family>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>زن</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>IRANSans</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>فرم افزودن مشخصات خودرو</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>IRANSans</family>
    </font>
   </property>
   <property name="text">
    <string>نکته: اطلاعات فرم را با دقت تکمیل کرده و بر روی دکمه ثبت اطلاعات کلیک کنید</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>140</y>
     <width>240</width>
     <height>37</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>IRANSans</family>
        <pointsize>10</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>نام و نام خانوادگی مالک</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="textFullName">
      <property name="font">
       <font>
        <family>IRANSans</family>
        <bold>false</bold>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>191</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <family>IRANSans</family>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>کد ملی</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="textNationalCode">
      <property name="font">
       <font>
        <family>IRANSans</family>
        <bold>false</bold>
       </font>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>180</y>
     <width>191</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="font">
       <font>
        <family>IRANSans</family>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>آدرس</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="textAddress">
      <property name="font">
       <font>
        <family>IRANSans</family>
        <bold>false</bold>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>270</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_9">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>IRANSans</family>
        <pointsize>10</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>نام خودرو</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboCarName">
      <property name="font">
       <font>
        <family>IRANSans</family>
        <bold>false</bold>
       </font>
      </property>
      <item>
       <property name="text">
        <string>پژو</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>پراید</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>سمند</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>پیکان</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>300</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_9">
      <property name="font">
       <font>
        <family>IRANSans</family>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>تعداد سرنشین</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spinCount">
      <property name="font">
       <font>
        <family>IRANSans</family>
        <bold>false</bold>
       </font>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>340</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_8">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>IRANSans</family>
        <pointsize>10</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>مدل</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="textYear">
      <property name="font">
       <font>
        <family>IRANSans</family>
        <bold>false</bold>
       </font>
      </property>
      <property name="inputMask">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>360</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="label_10">
      <property name="font">
       <font>
        <family>IRANSans</family>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>میزان سلامت خودرو</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spinHealth">
      <property name="font">
       <font>
        <family>IRANSans</family>
        <bold>false</bold>
       </font>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>400</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>IRANSans</family>
        <pointsize>10</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>رنگ</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnColor">
      <property name="font">
       <font>
        <family>IRANSans</family>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>انتخاب کنید</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>450</y>
     <width>211</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLabel" name="label_11">
      <property name="font">
       <font>
        <family>IRANSans</family>
        <pointsize>10</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>محل صدور پلاک</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTreeView" name="treePlace">
      <property name="font">
       <font>
        <family>IRANSans</family>
        <bold>false</bold>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
